Remove User ID and Password for service principal

kurtsung opened the pull request:
Remove User ID and Password from connection string when using service principal authentication is used.

PRmerger commented:
@padmavc : Thanks for your contribution to the Azure documentation! The author, @msftman, has been notified to review your proposed change.

padmavc commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

padmavc commented:
The author of the article is on 3 weeks’ vacation.  Erik, author manager, has provided sign off on behalf of him.  If that is not sufficient, suggest me what would be the best way to make corrections on the doc.


Thanks
Padma

From: PRmerger [mailto:notifications@github.com]
Sent: Tuesday, August 2, 2016 4:33 PM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Padma Chilakapati <padmavc@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Azure/azure-content] Added EDIFACT link and removed coming soon tag (#7165)


Only the author of the article can use the sign off comment to add the ready-to-merge label.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-content/pull/7165#issuecomment-237082244>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQPnLzKjU63jocFZwFbBVZ6cEhXiFbNpks5qb9PBgaJpZM4JZwJy>.

Erikre commented:
I'll #sign-off while @msftman is OOF.

ggailey777 commented:
PRMerger is a bot...not so good at conversation 😉 I will merge.

kurtsung committed:
Remove User ID and Password for service principle

Remove User ID and Password from connection string when using service principle authentication is used.

kurtsung committed:
Remove User ID and Password for service principle

Remove User ID and Password from connection string when using service principle authentication is used.

kurtsung committed:
Merge remote-tracking branch 'origin/patch-2'
